jdaviz/core/helpers.py

```diff
@@ -671,6 +672,7 @@ def default_viewer(self):
         (e.g., "imviz-0" or "cubeviz-0")."""
         return self._default_viewer.user_api
 
+    @deprecated(since="4.0", alternative="viz.plugins['Subset Tools'].import_region")
```

Comment:
When a change log is finally added, it needs to clarify that `max_num_regions=20` is no longer automatically set if loading image regions from file.